### DISCUSSION TITLE
Add a container ID setting without having to go to >advanced

### DISCUSSION THREAD

**Discussion by meemkade** [community-scripts/ProxmoxVE#134] [Ideas]
I think that it would be nice to have the scripts automatically ask you for what you want the LXC/VM id to be while your using it. I know that you can specify what ID for it to use inside of the advanced menu, but If you only wanted to set the ID, and change nothing else, you need to go through the whole advanced menu.
Having just a single prompt before it creates it by default asking what ID you want would be nice, and if you just hit enter or dont put in anything, like in the advanced section, it will just chose the next available ID.

**Comment by Mellowlynx** [comment]
That's where the advanced option is for, so you are not asked questions if you want to use the defaults.
